package.json

```diff
@@ -42,6 +42,7 @@
     "pre-push": "yarn lint"
   },
   "resolutions": {
+    "@metamask/snaps-sdk": "4.0.1",
```

Comment:
Why do we need this resolution? This seems to indicate a misalignment between dependencies and so it would be great if we can avoid this because it means that clients may need to apply the same resolution in order to use the packages in this repo.

Reply:
This can be removed. I originally thought this would help fix a type issue with the snaps sdk, the new version fixed it.